packages/paste-core/components/heading/__tests__/__snapshots__/heading.test.tsx.snap

```diff
@@ -3,7 +3,8 @@
 exports[`Heading it should render an H1 at fontSize90 1`] = `
 .emotion-1 {
   font-family: 'Whitney SSm A','Whitney SSm B','Helvetica Neue',Helvetica,Arial,sans-serif;
-  line-height: 1.15;
+  font-size: 0.875rem;
```

Comment:
Shouldn't these font-size tests be different font-sizes? Looks like they're all `0.875rem`.

Reply:
It's the base font size from the theme provider which we wrap every component in. It threw me too for an hour or so until I realised what I'd changed.

Reply:
Right. I see that now.